io/EptReader.cpp

```diff
@@ -121,32 +161,30 @@ void EptReader::initialize()
     m_queryBounds = m_args.bounds();
     handleOriginQuery();
 
-    const auto& scale(m_info->scale());
-    const auto& offset(m_info->offset());
-
     log()->get(LogLevel::Debug) << "Query bounds: " << m_queryBounds <<
         std::endl;
-    log()->get(LogLevel::Debug) << "Scale: " <<
-        scale[0] << ", " << scale[1] << ", " << scale[2] << std::endl;
-    log()->get(LogLevel::Debug) << "Offset: " <<
-        offset[0] << ", " << offset[1] << ", " << offset[2] << std::endl;
     log()->get(LogLevel::Debug) << "Threads: " << m_pool->size() << std::endl;
 }
 
 void EptReader::handleOriginQuery()
 {
     if (m_args.origin().empty()) return;
 
+    if (m_info->sources() == 0)
```

Comment:
The idea behind the count for `sources` was so that it could be optional to include this information in an EPT set.  I am thinking now that this key should be removed and that a valid EPT set must have a `ept-sources/list.json` which may be an empty array to omit the source information.